feature-settings-impl/src/main/java/io/novafoundation/nova/feature_settings_impl/domain/NetworkManagementChainInteractor.kt

```diff
@@ -62,7 +65,8 @@ interface NetworkManagementChainInteractor {
 class RealNetworkManagementChainInteractor(
     private val chainRegistry: ChainRegistry,
     private val nodeHealthStateTesterFactory: NodeHealthStateTesterFactory,
-    private val chainRepository: ChainRepository
+    private val chainRepository: ChainRepository,
+    private val accountInteractor: AccountInteractor
```

Comment:
In our current layers structure interactor should not depend on another interactor. Looks like it will be enough to just use `AccountRepository`

Reply:
Yes, agree, but I thought that it's not a repository responsibility to switch account if selected account  was deleted, so I used Interactor.

Reply:
Yeah indeed. But still we are moving to direction where interactor is only responsible for a certain screen (or flow)
What you are descriving sounds like a `UseCase` in our current terminology
Anyways  this is not so important